### ISSUE TITLE
Ensure Player API works

### ISSUE THREAD

**Issue by cjcolvar** [avalonmediasystem/avalon#2481]
## Description

We need to verify the player API continues to work. 

## Done looks like 

- [x] verification
- [x] issues created and prioritized

**Comment by cjcolvar** [comment]
https://wiki.dlib.indiana.edu/display/VarVideo/Player+API

**Comment by cjcolvar** [comment]
This could be tested by modifying this wiki page (https://wiki.dlib.indiana.edu/display/VarVideo/Test+Embed) and making sure everything described in the player api wiki page works.

**Comment by bkeese** [comment]
@yingfeng-iu did you get this email?

**Comment by yingfeng-iu** [comment]
Hi Adam, after adding currentPlayer I'm now getting new TypeError: currentPlayer is null, which means mejs4AvalonPlayer.player is not initiated, or the type is wrong. I see that in mejs2, currentPlayer =  AvalonPlayer.player, which has type MediaElementPlayer, in mejs4, I was setting it to MEJSPlayer.player, which also is MediaElementPlayer instance. it's assigned in MEJSPlayer.initializePlayer. I'm going to debug it a bit, if you have suggestions please let me know. thanks!

**Comment by adamjarling** [comment]
@yingfeng-iu Could this be a timing issue (maybe when the currentPlayer is set the player is not yet initialized?)  Or is the problem the iFrame window context?   I need to split for the day, but if you want to screen share tomorrow morning and show me what's going on maybe that'd help.

**Comment by yingfeng-iu** [comment]
Adam/Brian, thanks for helping!

Populating the currentPlayer inside success event didn't work, that method didn't get called during loading of the page. I ended up adding it inside the mejs4 player initialize method, but also have to declare it as global var in the script files (otherwise it's not visible outside the class).
Tested locally, all functions work except that toggle loop doesn't seem to have any effect. Maybe we need to turn on some mejs4 setting for that?
